packages/modules/web_themes/koala/source/src/stores/mqtt-store.ts

```diff
@@ -2576,10 +2736,12 @@ export const useMqttStore = defineStore('mqtt', () => {
     chargePointStateMessage,
     chargePointFaultState,
     chargePointFaultMessage,
+    DcChargingEnabled,
```

Comment:
Fällt Dir hier etwas auf? Alle anderen Exporte fangen mit einem kleinen Buchstaben an, da es Methoden sind.
```suggestion
    dcChargingEnabled,
```

Reply:
korrigiert